internal/config/config.go

```diff
@@ -192,7 +192,7 @@ type AzureConfig struct {
 	IDKeyDigest Digests `yaml:"idKeyDigest" validate:"required_if=EnforceIdKeyDigest true,omitempty"`
 	// description: |
 	//   Enforce the specified idKeyDigest value during remote attestation.
-	EnforceIDKeyDigest *bool `yaml:"enforceIdKeyDigest" validate:"required"`
+	EnforceIDKeyDigest idkeydigest.EnforceIDKeyDigest `yaml:"enforceIdKeyDigest" validate:"required"`
```

Comment:
So we either need to implement a custom validator that maps from bool to enum or add a docs migration section. I would vote for the custom validator.

Reply:
https://github.com/edgelesssys/constellation/blob/c2b11777c864c44f971c2fced278b4e24bce93e4/internal/attestation/idkeydigest/idkeydigest.go#L69

Already implemented